bot/seasons/evergreen/game.py

```diff
@@ -198,42 +208,38 @@ def __init__(self, bot: Bot):
     async def get_pages(self, data: List[Dict[str, Any]]) -> List[Tuple[str, str]]:
         """Generate all game pages, do additional requests to IGDB API."""
         pages = []
-        for game in data:
-            pages.append(await self.create_page(game))
+        [pages.append(await self.create_page(game)) for game in data]
         return pages
 
     async def create_page(self, data: Dict[str, Any]) -> Tuple[str, str]:
         """Create content of Game Page."""
         # Create page content variable, what will be returned
         page = ""
 
-        # Get keys to one variable. Used for code style
-        keys = data.keys()
-
         # If game have cover, generate URL of Cover, if not, let url empty
-        if 'cover' in keys:
+        if 'cover' in data:
             url = f"{IMAGE_BASE_URL}t_cover_big/{data['cover']['image_id']}.jpg"
         else:
             url = ""
 
         # Add title with hyperlink and check for storyline
         page += f"**[{data['name']}]({data['url']})**\n"
-        page += data['summary'] + "\n\n" if 'summary' in keys else "\n"
+        page += data['summary'] + "\n\n" if 'summary' in data else "\n"
 
         # Add release date if key is in game information
-        if 'first_release_date' in keys:
+        if 'first_release_date' in data:
             page += f"**Release Date:** {datetime.utcfromtimestamp(data['first_release_date']).date()}\n"
 
         # Add other information
-        page += f"**Rating:** {'{0:.2f}'.format(data['total_rating']) if 'total_rating' in keys else '?'}/100 "
-        page += f":star: (based on {data['total_rating_count'] if 'total_rating_count' in keys else '?'})\n"
+        page += f"**Rating:** {'{0:.2f}'.format(data['total_rating']) if 'total_rating' in data else '?'}/100 "
+        page += f":star: (based on {data['total_rating_count'] if 'total_rating_count' in data else '?'})\n"
 
         page += f"**Platforms:** "
-        page += f"{', '.join(pf['name'] for pf in data['platforms']) if 'platforms' in keys else '?'}\n"
+        page += f"{', '.join(pf['name'] for pf in data['platforms']) if 'platforms' in data else '?'}\n"
 
-        page += f"**Status:** {GameStatus(data['status']).name if 'status' in keys else '?'}\n"
+        page += f"**Status:** {GameStatus(data['status']).name if 'status' in data else '?'}\n"
 
-        if 'age_ratings' in keys:
+        if 'age_ratings' in data:
             rating = f"""{', '.join(AgeRatingCategories(age['category']).name + ' ' + AgeRatings(age['rating']).name
```

Comment:
Why does this need an f-string wrapped around it?  join returns a string. I'd use an f-string inside the join, but not around it.
```
rating = ', '.join(
    f'{AgeRatingCategories(age["category"]).name} {AgeRatings(age["rating"]).name}'
    for age
    in date['age_ratings']
)
```